test/fixtures/schema.sql

```diff
@@ -305,7 +305,7 @@ SET search_path = postgrest, pg_catalog;
 
 CREATE TABLE auth (
     id character varying NOT NULL,
-    rolname name NOT NULL,
+    rolname name NOT NULL DEFAULT 'postgrest_test_author',
```

Comment:
Weird, yes it does.

Ah, that's because there is no constraint on the SQL schema for values. I
can throw in the function from the wiki and add a test case.
On Wed, Sep 23, 2015 at 11:26 PM Joe Nelson <notifications@github.com>
wrote:

> In test/fixtures/schema.sql
> <https://github.com/begriffs/postgrest/pull/291#discussion_r40282580>:
>
> > @@ -305,7 +305,7 @@ SET search_path = postgrest, pg_catalog;
> >
> >  CREATE TABLE auth (
> >      id character varying NOT NULL,
> > -    rolname name NOT NULL,
> > +    rolname name NOT NULL DEFAULT 'postgrest_test_author',
>
> Strangely when I remove the default value here the tests continue to pass.
> I was expecting a null violation. Does it do the same for you?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/begriffs/postgrest/pull/291/files#r40282580>.
>